crates/hdk/src/map_extern.rs

```diff
@@ -1,5 +1,37 @@
 use crate::prelude::*;
 
+#[macro_export]
+macro_rules! map_extern_preamble {
+    ( $guest_ptr:ident, $len:ident, $inner:ident, $input:ty, $output:ty ) => {
+        // Setup tracing.
+        // @TODO feature flag this?
```

Comment:
leftover or intentional?

Reply:
intentional, there's no feature flag for it currently